docs/.vuepress/config.js

```diff
@@ -39,25 +39,49 @@ module.exports = {
     logo: '/images/apos-dark.png',
     nav: [
       {
-        text: 'Guide',
-        link: '/guide/introduction.md'
+        text: 'v2 Docs',
+        link: 'https://v2.docs.apostrophecms.org'
```

Comment:
Actually, can we make this a "Versions" dropdown so you can see there are the two versions here?